StPicoDstMaker/StPicoDstMaker.cxx

```diff
@@ -575,7 +575,14 @@ Int_t StPicoDstMaker::MakeRead()
     LOG_WARN << " No input files ... ! EXIT" << endm;
     return kStWarn;
   }
-  mChain->GetEntry(mEventCounter++);
+  int bytes = mChain->GetEntry(mEventCounter++);
+  
+  while ( bytes <= 0 ){
+    if ( mEventCounter >= mChain->GetEntriesFast() )
+      return kStEOF;
+    bytes = mChain->GetEntry(mEventCounter++);
```

Comment:
Is this ok to silently skip weird events?

Reply:
38daa1488560b498ac8e06871e030f790a17a0f7

Reply:
Thanks